_posts/2018-04-03-CocoaPods-1.5.0.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "CocoaPods 1.5.0 &#8212; Swift Static Libraries"
+author: segiddins
```

Comment:
You not claiming it? :P

Reply:
ha, @segiddins had written the first draft, I just took it and made a PR with a few adjustments